src/test/java/com/exasol/adapter/document/files/extension/OldExtensionIT.java

```diff
@@ -0,0 +1,195 @@
+package com.exasol.adapter.document.files.extension;
```

Comment:
Shouldn't this file be deleted?

Reply:
Not yet. Once we have a new release we can delete this. The extension upgrade tests must be custom until then.